bzira.py

```diff
@@ -173,21 +166,19 @@ def bz_to_jira_version(options, bug):
      'normal' : 'Major',
      'minor' : 'Minor',
      'trivial' : 'Trivial',
-     'enhancement' : 'Trivial' #TODO: is bz enhancement really indictor for type is feature or is it purely a priority/complexity flag ?
+     'enhancement' : 'Trivial' #TODO: determine if Bugzilla's 'enhancement' is really an indicator of a feature request, or simply a priority/complexity flag
     }
-
     
 def bz_to_jira_priority(options, bug):
     return bz2jira_priority[bug.severity] # jira is dumb. jira priority is severity.
 
-
 bz2jira_status = {
            "NEW" : "Open",
            "REOPENED": "Reopened",
            "RESOLVED" : "Resolved",
            "VERIFIED" : "Verified",
            "CLOSED" : "Closed",
-           "ASSIGNED" : "Coding In Progress", # is this the right approximation ?
+           "ASSIGNED" : "Coding In Progress", # TODO: determine if this is a valid approximation
```

Comment:
this is a valid approximation, but it is not necessarily the right one. Please don't change the meaning to something different.